src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js

```diff
@@ -139,6 +140,15 @@ const TriggerFormChannels = ({
           <FormikLabel text='Notify me via' />
         </SidecarExplanationTooltip>
         <div className={styles.notifyBlock}>
+          <ChannelCheckbox
```

Comment:
Can we go through array of channels here?

Reply:
We can